pandas/tests/arrays/test_array.py

```diff
@@ -182,6 +182,13 @@ def test_array_copy():
     assert np.shares_memory(a, b._ndarray) is True
 
 
+def test_array_from_series():
+    # GH38543
+    np.array([pd.Series({0: 0})])
```

Comment:
Could you use `pytest.mark.parameterize` to test each case?

Additionally, these tests should follow the format

```
result = np.array(np.array([pd.Series({0: 0})]))
expected = np.array(...) # another way to construct the result
tm.assert_numpy_array_equal(result, expected)
```